contracts/interfaces/IDCAHubCompanion.sol

```diff
@@ -29,6 +29,35 @@ interface IDCAHubCompanionLibrariesHandler {
   function secondsUntilNextSwap(IDCAHub hub, Pair[] calldata pairs) external view returns (uint256[] memory);
 }
 
+interface IDCAHubCompanionTakeWithdrawAndSwapHandler {
```

Comment:
Why the name `TakeWithdrawAndSwap`? What is it withdrawing? What is it swapping? There is something missing, or this is not the right naming.

Reply:
Initially, `sendToRecipient` was called `Withdraw`, that's why. Also, I'll be adding Swap extensions to the contract, that the "Swap" part

Would you prefer I call it `TakeSendAndSwapHandler`?

Reply:
Oh. Ok. Makes sense now haha. Sure. I'd take your `TakeSendAndSwapHandler`